constants/index.js

```diff
@@ -16,5 +16,31 @@ module.exports = {
     'EQU',
     'NHP'
   ],
-  readableDateFormat: 'DD MMMM YYYY'
+  readableDateFormat: 'DD MMMM YYYY',
+  accreditationBodies: [
+    'Society of Biology',
+    'Universities Group',
+    'Scottish Accreditation Board',
+    'Royal Veterinary College',
+    'Other'
+  ],
+  moduleCodes: [
+    'L',
+    'E1',
+    'PILA(theory)',
+    'PILA(skills)',
+    'PILB',
+    'PILC',
+    'K(theory)',
+    'K(skills)',
+    'E2',
+    'PPL'
+  ],
+  procedureOptions: [
+    'A. Minor/minimally invasive procedures not requiring sedation, analgesia or general anaesthesia',
+    'B. Minor/minimally invasive procedures involving sedation...',
+    'C. Surgical procedures...',
+    'D. Use of neuro...',
+    'F. Other'
+  ]
```

Comment:
I think we should look to store the _data_ for these purely as `A`, `B`, `C` etc, and then store the descriptions in content.

Reply:
Also, where we're adding new constants that might/will be used across different places, we probably want to keep them in https://github.com/UKHomeOffice/asl-constants